src/sources/AdwordsReports.ts

```diff
@@ -133,13 +133,21 @@ export class AdwordsReports extends EtlSource {
     return this.currentSavePoint['startDate'];
   }
 
+  /**
+   * get the sql period to use in the query
+   * Overwrite this to use different period
+   */
+  protected getSqlQueryPeriod(): string {
+    return `DURING ${this.currentSavePoint['startDate']},${this.currentSavePoint['startDate']}`;
```

Comment:
using both startDate here. is it correct?

Reply:
Yes this goes one day at the time